plugins/gatsby-source-nav/gatsby-node.js

```diff
@@ -113,34 +105,27 @@ const createWhatsNewNav = async ({ createNodeId, nodeModel }) => {
     },
   });
 
-  const now = new Date();
-  const firstOfMonth = startOfMonth(now);
-  const lastMonth = sub(firstOfMonth, { months: 1 });
+  const currentYear = new Date().getFullYear();
+  const postsByYear = groupBy(posts, (post) => parseDate(post).getFullYear());
+  const thisYearsPosts = postsByYear.get(currentYear) || [];
 
-  const thisMonthsPosts = posts.filter((post) =>
-    isEqualOrAfter(parseDate(post), firstOfMonth)
+  const postsByMonth = groupBy(thisYearsPosts, (post) =>
+    parseDate(post).toLocaleString('default', { month: 'long' })
   );
 
-  const lastMonthsPosts = posts.filter(
-    (post) =>
-      isEqualOrAfter(parseDate(post), lastMonth) &&
-      isBefore(parseDate(post), firstOfMonth)
+  const previousYearsPosts = Array.from(postsByYear.entries()).filter(
```

Comment:
QQ, do we need `Array.from` if we're using `.entries()`?

Reply:
Yes because `.entries()` returns an `Iterator` unfortunately and not an array. I unfortunately made this mistake a wasted a `yarn start` cycle due to not wrapping this in `Array.from` 😅